corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -370,6 +370,9 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                 autoUpdateInput: false,
                 "autoApply": true,
             });
+            this.ui.dateRange.attr("placeholder", dateFormat + separator + dateFormat);
```

Comment:
I think we should redefine `dateFormats` to prioritize `dateFormat` in the list of allowable formats if `dateFormat` is the recommended input pattern.

This is because we parse the input of `dateRange` using  `dateFormats` as order of priority input formats. `parts = _.map(oldValue.split(separator), function (v) { return moment(v, dateFormats);`

Reply:
Fair point, just updated it.  I'd also forgotten about that part that parses input formats for the date range picker.  This means we can consistently apply the input parsing to both widgets!  Working on that now

Reply:
That's a win for consistent date picker behavior!

I thought you were running into issues previously with using strict mode on date range picker based on this comment. But I now realize I think you were looking at something else https://github.com/dimagi/commcare-hq/pull/32628#issuecomment-1460860379

Reply:
I was, because I was trying to do it in the library (as the single date picker does with `parseInputDate`).  Turns out we already interpret the dates in the date range picker in this code though, so i can just jump in there.  I did a bunch more fiddling to unify input processing.  I still need to do more testing, but let me know what you think.